pkg/server/apiextensions.go

```diff
@@ -144,27 +142,23 @@ func (p *systemCRDProvider) Keys(ctx context.Context, clusterName logicalcluster
 	return sets.NewString()
 }
 
-// apiBindingAwareCRDLister is a CRD lister combines APIs coming from APIBindings with CRDs in a workspace.
-type apiBindingAwareCRDLister struct {
-	kcpClusterClient  kcpclientset.ClusterInterface
-	crdLister         apiextensionslisters.CustomResourceDefinitionLister
-	workspaceLister   tenancylisters.ClusterWorkspaceLister
-	apiBindingLister  apislisters.APIBindingLister
-	systemCRDProvider *systemCRDProvider
+// APIBindingAwareCRDLister is a CRD lister combines APIs coming from APIBindings with CRDs in a workspace.
+type APIBindingAwareCRDLister struct {
```

Comment:
used outside of the package?

Reply:
I don't remember why I exported it. Fixing